### PR TITLE
Add CVE-2021-29484 - Ghost CMS DOM XSS

### PR DESCRIPTION
**Pull request by rootxharsh**
### Template / PR Information

Ghost CMS DOM XSS - https://github.com/TryGhost/Ghost/security/advisories/GHSA-9fgx-q25h-jxrg

- Added CVE-2021-29484
- References: https://github.com/TryGhost/Ghost/security/advisories/GHSA-9fgx-q25h-jxrg

### Template Validation

I've validated this template locally?
- YES

### CODE REVIEW

**Comment by DhiyaneshGeek** [comment]
@rootxharsh is this just a detection part ?

**Comment by rootxharsh** [comment]
Yes

File: cves/2021/CVE-2021-29484.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  author: rootxharsh, iamnoooob
+  severity: medium
+  tags: ghost
+  reference: https://github.com/TryGhost/Ghost/security/advisories/GHSA-9fgx-q25h-jxrg
+
+requests:
+  - raw:
+      - |
+        GET /ghost/preview HTTP/1.1
+        {{Hostname}}
```
Comment (toufik-airane) [inline cves/2021/CVE-2021-29484.yaml:14]:
```suggestion
  - method: GET
    path:
      - "{{BaseURL}}/ghost/preview"
```

**Review by toufik-airane — CHANGES_REQUESTED** [review]
Dear @rootxharsh,

Nice contribution. I'm not a fan of raw requests when it comes to a simple GET request that not involving body or special headers.

What do you think to use the `method: GET` instead of the `raw`?
Moreover, you probably forget to put the `Host:` in front of the `{{Hostname}}`.

Thank you.

File: cves/2021/CVE-2021-29484.yaml (hunk 1 above)
Comment (rootxharsh) [reply cves/2021/CVE-2021-29484.yaml]: Nice call!

**Pull request merged**